Avoid test_pretty_no_wrap_line failure on large screens

See https://github.com/sympy/sympy/pull/28411

Author: Alexander Grund (TU Dresden)
diff --git a/sympy/printing/pretty/tests/test_pretty.py b/sympy/printing/pretty/tests/test_pretty.py
--- a/sympy/printing/pretty/tests/test_pretty.py
+++ b/sympy/printing/pretty/tests/test_pretty.py
@@ -5344,8 +5344,8 @@ def test_pretty_no_wrap_line():
     huge_expr = 0
     for i in range(20):
         huge_expr += i*sin(i + x)
-    assert xpretty(huge_expr            ).find('\n') != -1
-    assert xpretty(huge_expr, wrap_line=False).find('\n') == -1
+    assert xpretty(huge_expr, num_columns=80            ).find('\n') != -1
+    assert xpretty(huge_expr, num_columns=80, wrap_line=False).find('\n') == -1
 
 
 def test_settings():
